pay-api/src/pay_api/services/statement.py

```diff
@@ -128,68 +127,6 @@ def asdict(self):
         d = statement_schema.dump(self._dao)
         return d
 
-    @staticmethod
-    def get_payment_methods_query(statement: StatementModel):
```

Comment:
Wont work for payments that don't show up in the `payment_account_history` table